extensions/wikia/MyHome/MyHome.class.php

```diff
@@ -181,14 +181,18 @@ public static function getInitialMainPage(Title &$title) {
 		//this is not used for Corporate Sites where Wikia Visualization is enabled
 		if( $wgUser->isLoggedIn() && empty($wgEnableWikiaHomePageExt) ) {
 			$value = $wgUser->getGlobalPreference(UserPreferencesV2::LANDING_PAGE_PROP_NAME);
-			switch($value) {
-				case UserPreferencesV2::LANDING_PAGE_WIKI_ACTIVITY:
-					$title = SpecialPage::getTitleFor('WikiActivity');
-					break;
-				case UserPreferencesV2::LANDING_PAGE_RECENT_CHANGES:
-					$title = SpecialPage::getTitleFor('RecentChanges');
-					break;
+			if ( $value == UserPreferencesV2::LANDING_PAGE_WIKI_ACTIVITY ) {
```

Comment:
Not particularly. But I need more customized condition than simple switch case. Other option was to use if-else inside a switch case, which didn't seem to follow a specific standard. So I choose to change the who statement.